hr_holidays_public/models/hr_holidays_public.py

```diff
@@ -55,15 +55,17 @@ def name_get(self):
 
     @api.model
     @api.returns("hr.holidays.public.line")
-    def get_holidays_list(self, year, employee_id=None):
+    def get_holidays_list_by_year(self, years, employee_id=None):
```

Comment:
You are still changing the API... Please check all my comments and my explanations, and ask me for any doubt, but let's try to avoid loops.